src/index.ts

```diff
@@ -41,9 +41,6 @@ export function importCppBindingsModule(): PrivateV8CpuProfilerBindings {
   // This is for cases where precompiled binaries were not provided, but may have been compiled from source.
   if (platform === 'darwin') {
     if (arch === 'x64') {
-      if (abi === '93') {
-        return require('./sentry_cpu_profiler-darwin-x64-93.node');
-      }
```

Comment:
This PR removes all references to the 93 abi binaries because we no longer build them and they were causing the new tests to fail!

Reply:
should this be a separate PR, where we stop building those? This does not seem to be a test-only change here? Or is it just not used anymore anywhere?

Reply:
> should this be a separate PR, where we stop building those? 

We haven't been building these binaries for a while. They are from Node 16. The new tests simply highlighted that these requires are historic and needed cleaning up.